stereo_image_proc/launch/stereo_image_proc.launch.py

```diff
@@ -224,7 +226,15 @@ def generate_launch_description():
                     PythonLaunchDescriptionSource([
                         FindPackageShare('image_proc'), '/launch/image_proc.launch.py'
                     ]),
-                    launch_arguments={'container': LaunchConfiguration('container')}.items()
+                    launch_arguments={
+                        'container': PathJoinSubstitution([
```

Comment:
I don't think we should use PathJoinSubstitution since it is intended for joining system paths, not ROS namespaces. E.g. on Windows, I presume it uses a different separator (`\`).

Reply:
Noted. We don't need this anymore if we prepend earlier as you suggested